generate_response.py

```diff
@@ -995,11 +1035,22 @@ def HumanEval_experiment(dataset, dataset_loc, option, model, topn, temperature,
                 prompt = create_prompt(description, option, remove_percentage)
                 if option.startswith('randRemove'):
                     # legacy part
+                    # Dont need to make any changes to this I think, since this is legacy part
                     response_list, code_list, qq_list = description_2_code_one_round(prompt, model, topn, temperature, args, open_source_model, tokenizer)
                 else:
                     original_prompt = problem['prompt']
-                    prompt_start = ORIGINAL_PROMPT_START_0 if input_prompt == 'prompt' else PROMPT_START_3_v2
-                    response_list, code_list, qq_list, ans_list = description_2_code_multi_rounds(prompt_start, description, original_prompt, model, topn, temperature, args, open_source_model, tokenizer, cached_responses.get(key, ''), cached_qqs.get(key, 0), cached_answers.get(key, ''))
+                    entry_point = problem['entry_point']
+                    task_id = problem['name']
+                    # prompt_start = ORIGINAL_PROMPT_START_0 if input_prompt == 'prompt' else PROMPT_START_3_v2
+                    # We will use "Prompt_Modified" to check whether AgentCoder is getting a modified prompt or an original prompt, based on which, we decide whether to send in a "generate clarifying questions" prompt or not.
+                    # A new prompt called PROMPT_START_3_v4 has been created for the same.
+                    Prompt_Modified = False
+                    if input_prompt == 'prompt':
+                        prompt_start = ORIGINAL_PROMPT_START_0
+                    else:
+                        prompt_start = PROMPT_START_3_v2
+                        Prompt_Modified = True
```

Comment:
use `prompt_modified = False if input_prompt == 'prompt' else True`

Reply:
change Prompt_Modified  to lower case for code convention

Reply:
done.